data_steward/cdr_cleaner/cleaning_rules/deid/registered_cope_survey_suppression.py

```diff
@@ -37,6 +37,8 @@
 |1310144                        | https://athena.ohdsi.org/search-terms/terms/1310144           |
```

Comment:
Suggestion: Since the list of concepts, is moved to a CSV file, we can remove this table and refer to the table path in the doc string.